src/global.d.ts

```diff
@@ -1,2 +1,14 @@
 /// <reference types="@sveltejs/kit" />
-interface Window {}
+interface Window { }
+
+namespace App {
+    interface Error {
+        type: string = 'unknown',
```

Comment:
doesn't seem like a valid syntax.
maybe also make it nullable to not force a `type` on errors that are not `AppwriteException`s.